IPL/src/processes/IPLFrequencyFilter.cpp

```diff
@@ -51,7 +51,12 @@ void IPLFrequencyFilter::destroy()
 
 bool IPLFrequencyFilter::processInputData(IPLImage* data , int, bool)
 {
-    _input = new IPLComplexImage(*data->toComplexImage());
+    IPLComplexImage* complexImageData = data->toComplexImage();
+    if (NULL == complexImageData) {
+        // TODO write an error message
```

Comment:
Here and below, perhaps it would be better to inform the user that this connection is not made, but I saw that they way you do it is through `MainWindow::showMessage` and `MainWindow::logMessage`, but those are not exposed to the dll.